mappings/net/minecraft/block/TransparentBlock.mapping

```diff
@@ -1,3 +1,7 @@
 CLASS net/minecraft/class_46 net/minecraft/block/TransparentBlock
+	FIELD field_178 transparent Z
```

Comment:
Is this public? If not, does it have an accessor? In either case, the public-facing member possibly should be prefixed with `is` in traditional Java style